#1436070, #1436069: Remove inlined CSS, fix semantic markup

andymikulski opened the pull request:
- Fixes [#1436070 (non-semantic markup)](https://bugzilla.mozilla.org/show_bug.cgi?id=1436070)
- Fixes [#1436069 (inlined CSS)](https://bugzilla.mozilla.org/show_bug.cgi?id=1436069)

**Notes:**
- Instead of replacing all of the bold markup with something like `<span class="bold">`, I decided to simply unify all of the markup to use the same element - `b`.
  - It turns out that some of the javascript looks for `b` tags when operating. The compromise was to simply unify the markup that's used for bold text so the styles at least stem from the same definition.
- A couple of files missed the CSS->LESS conversion, those are included here ase they were affected by the `strong/b/.bold` change
- Inlined styles are replaced with classes. The `debug_login.less` file was created as a result, which is included.

**To review:**
- Build the site, navigate the site, double check the LESS. Should be pretty straight-forward.

Osmose commented on webapp-django/crashstats/api/static/api/css/documentation.less:
```diff
@@ -1,24 +1,36 @@
+@import "@{root-path}/base/variables.less";
+
+p {
+    &.url {
```
This could also just be `p.url` since it's the only thing in the block. Is this an explicit style choice?

I'm not against it or anything, I just noticed it and was curious.

andymikulski replied:
I think I just converted the code in the file from top-to-bottom and simply didn't notice that this selector didn't contain much. I've collapsed it down to `p.url`.

Osmose commented on webapp-django/crashstats/base/jinja2/crashstats_base.html:
```diff
@@ -176,8 +176,7 @@ <h1>Product Navigation</h1>
                         id="signin2"
-                        class="google-signin"
-                        style="float:right"
+                        class="google-signin float_right"
```
Could we maybe just apply the float to the `google-signin` class, or add a different class? `float_right` isn't a terribly semantic classname.

Osmose commented on webapp-django/crashstats/profile/jinja2/profile/profile.html:
```diff
@@ -34,7 +34,7 @@ <h2>Your Profile</h2>
 
             <!-- clear after two floats -->
-            <div style="clear: both;"></div>
+            <div class="clear"></div>
```
Could we use something like [this clearfix](https://css-tricks.com/snippets/css/clear-fix/) to have the container clear the links and remove the need for this element altogether?

andymikulski commented:
>I can't figure out why that test error is resulting from your changes, ...

The issue was a static file (`analyze-model-fetches.css`) was being referenced, however I changed the extension to `.less`. I've changed the entire `analyze-model-fetches.html` file to use bundles, which should resolve the issue. Notably, this is also the last of the JS/LESS that was _not_ already in bundles!

I've corrected the code review items and pushed the new bundle, so this should now properly pass CI.

andymikulski commented:
@Osmose This is all set, though I am unable to merge!

<img width="527" alt="screen shot 2018-04-02 at 4 33 25 pm" src="https://user-images.githubusercontent.com/2767162/38221292-98c38e2a-3693-11e8-923e-00293eb575fd.png">

Osmose commented:
Huh, I thought I granted you access. Weird. I'll see if I can fix that.

Osmose commented:
Yeah, you totally have write access:

![screen shot 2018-04-02 at 6 20 02 pm](https://user-images.githubusercontent.com/193106/38223968-867acef4-36a2-11e8-88fd-8c141ea3f017.png)

wtf?
